src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1,4 +1,27 @@
-// Add the required types and props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
-);
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+import { UserInfo } from '../UserInfo';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoInfo:React.FC<Props> = ({ todo }) => {
+  const { title, completed, user } = todo;
+
+  return (
+    <article
+      className={classNames(
+        'TodoInfo',
+        {
+          'TodoInfo--completed': completed,
+        },
+      )}
+    >
+
+      <h2 className="TodoInfo__title">{title}</h2>
+
+      {user && <UserInfo user={user} />}
```

Comment:
The temptation to provide one-line piece of code is really huge, we all been there. Still it is more readable when moved to the new line 😄 
```suggestion
      {user && (
        <UserInfo user={user} />
      )}
```

Reply:
Thank you Nazar, I fixed them.  Deciding where to make a new line and how to name things are harder than solving the tasks. :sweat_smile: